### PR TITLE
SAVAS - Pricing page exercise

### PR DESCRIPTION
**Pull request by savaspamuk**

### CODE REVIEW

File: responsive-grid-layout/savas/global.css [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+.sidebar-right {
+    background-color: #ffb100;
+}
+
+.footer {
+    background-color: #006c67;
+    color: white;
+}
+
+@media all and (max-width: 768px) {
```
Comment (annasvst) [inline responsive-grid-layout/savas/global.css:41]: For device size specific code (defined in media query selectors), you can use the "responsive.css" file. This separation would make the code for bigger projects more readable. 

Then, just make sure that you css files are added to the HTML file in the correct order: from more generic to more specific:
```
<link rel="stylesheet" href="./global.css">
<link rel="stylesheet" href="./responsive.css">

```

**Review by annasvst — CHANGES_REQUESTED** [review]
Looking good Savaş, I've added a few suggestions in the comments, please take a look and let me know if something is unclear :)

File: responsive-grid-layout/savas/global.css [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+    .container {
```
[the remark this answers is not in the document]
Comment (savaspamuk) [reply responsive-grid-layout/savas/global.css:42]: Ok, I deleted these three values from where I wrote them again.

File: responsive-grid-layout/savas/global.css [hunk 3]
```diff
@@ -0,0 +1,144 @@
+* {
+    box-sizing: border-box;
+    font-family: -apple-system,BlinkMacSystemFont,"Segoe UI",Roboto,"Helvetica Neue",Arial,sans-serif,"Apple Color Emoji","Segoe UI Emoji","Segoe UI Symbol";
```
[the remark this answers is not in the document]
Comment (savaspamuk) [reply responsive-grid-layout/savas/global.css:3]: Ok, I used the font-family property in only body.

File: responsive-grid-layout/savas/global.css [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+        height: 100vh;
+        display: grid;
+        grid-template-columns: 1fr;
+        grid-template-rows: 100px auto 300px;
+        grid-template-areas:
+        "header"
+        "content"
+        "footer";
+    }
+
```
[the remark this answers is not in the document]
Comment (savaspamuk) [reply responsive-grid-layout/savas/global.css:53]: Ohhh, it's true, I missed it. I deleted these codes from the media queries.

File: responsive-grid-layout/savas/global.css (hunk 1 above)
Comment (savaspamuk) [reply responsive-grid-layout/savas/global.css:41]: I got it and moved these codes to responsive.css file.

**Review by annasvst — CHANGES_REQUESTED** [review]
Well done,  Savaş!

**Review by annasvst — APPROVED** [review]

**Pull request merged**